Game/Assets/Scripts/src/Script/HelpMenuController.cpp

```diff
@@ -0,0 +1,132 @@
+#include "HelpMenuController.h"
+#include "Component/ComponentAudioSource.h"
+#include "Component/ComponentButton.h"
+
+#include "Main/Application.h"
+#include "Main/GameObject.h"
+#include "Module/ModuleInput.h"
+
+#include "MainMenuController.h"
+#include "UIMainMenuInputController.h"
+
+#include "imgui.h"
+
+HelpMenuController* HelpMenuControllerDLL()
+{
+	HelpMenuController* instance = new HelpMenuController();
+	return instance;
+}
+
+HelpMenuController::HelpMenuController()
+{
+	panel = new PanelComponent();
+}
+
+void HelpMenuController::Awake()
+{
+	audio_source = (ComponentAudioSource*)audio_controller->GetComponent(Component::ComponentType::AUDIO_SOURCE);
+	main_menu_controller = static_cast<MainMenuController*>(main_menu_panel->GetComponentScript("MainMenuController")->script);
```

Comment:
this can't be main_menu_controller but the previous scene otherwise you can't not use this script for example in the pause menu

Reply:
True that, I will change it!